distribution/docs/src/main/resources/_contents/_catalog-frameworks/catalog-fanout-framework-contents.adoc

```diff
@@ -0,0 +1,270 @@
+| ||| Catalog Provider | | Connected Source(s) || ||Federated Source(s)||| |
+| |||  cDEF            | |  cDEF               || || cDEF              ||| |
+| ||\------------------/ \---------------------/| |\-------------------/|| |
+| |+-------^----------------------^-------------+ +----^----------^-----+| |
+| +--------|----------------------|--------------------|----------|------+ |
+\----------|----------------------|--------------------|----------|--------/
+           |                      |                    |          \-------------------\
+           |                      |                    |                              |
+           v                      v                    v                              v
+/=-------------------\ /----------------------\ /---------=------------\     /----------------------\
+|Internal Data Source| |External Data Source 1| |External Data Source 3| ... |External Data Source N|
+\--------------------/ \--------------------=-/ \----------------------/     \-------=--------------/
+....
+
+In addition, the Fanout Catalog Framework provides the following benefits:
+
+* Backwards compatibility (e.g., federating with older versions) with existing older versions of ${branding}.
+* A single node being exposed from an enterprise, thus hiding the enterprise from an external client.
+* Ensures all queries and resource retrievals are federated.
+
+The Fanout Catalog Framework is a core class of ${branding} when configured as a fanout proxy.
+It provides the methods for query and resource retrieval operations on the Sources, where all operations are enterprise-wide operations.
+By contrast, the Standard Catalog Framework supports create/update/delete operations of metacards, in addition to the query and resource retrieval operations.
+
+Use the Fanout Catalog Framework if:
+
+* exposing a single node for enterprise access and hiding the details of the enterprise, such as federate source's names, is desired.
+* access to individual federated sources is not required.
+* access to a catalog provider to create, update, and delete metacards is not required.
+
+The Fanout Catalog Framework also maintains a list of `ResourceReaders` for resource retrieval operations.
+A resource reader is matched to the scheme (i.e., protocol, such as `file://`) in the URI of the resource specified in the request to be retrieved.
+
+Site information about the fanout configuration can be retrieved using the Fanout Catalog Framework.
+Site information includes the source's name, version, availability, and the list of unique content types currently stored in the source (e.g., NITF).
+Details of the individual federated sources is not included, only the fanout catalog framework.
+
+===== Installing the Fanout Catalog Framework
+
+The Fanout Catalog Framework is bundled as the `catalog-core-fanoutframework` feature and can be installed and uninstalled using the normal processes described in Configuration.
```

Comment:
we can remove this section, should be covered in the Catalog Framework documentation.